--- /usr/local/bin/mvim	2017-04-05 12:14:09.000000000 -0400
+++ mvim	2017-04-07 17:02:49.000000000 -0400
@@ -83,6 +83,7 @@
 	# terminal instead of the console log.
 	# But if you use open instead, you will need to fully qualify the
 	# path names for any filenames you specify, which is hard.
+    [[ "x$tab" != "x" ]] && opts="$opts --servername $tab --remote-tab-silent"
 	exec "$binary" -g $opts ${1:+"$@"}
 else
 	exec "$binary" $opts ${1:+"$@"}
